packages/react/ds/src/alert/alert.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+import { useState, type ReactNode } from 'react';
+import { tv, type VariantProps } from 'tailwind-variants';
+import { Icon } from '../icon/icon.js';
+import { IconButton } from '../icon-button/icon-button.js';
+
+const alertVariants = tv({
+  slots: {
+    base: 'gi-alert-base',
+    baseDismissible: 'gi-alert-base-dismissible',
+    container: 'gi-alert-container',
+    heading: 'gi-alert-title',
+    dismiss: 'gi-alert-dismiss',
+  },
+  variants: {
+    variant: {
+      info: {
+        base: 'gi-alert-info',
+        baseDismissible: 'gi-alert-info',
+      },
+      danger: {
+        base: 'gi-alert-danger',
+        baseDismissible: 'gi-alert-danger',
+      },
+      success: {
+        base: 'gi-alert-success',
+        baseDismissible: 'gi-alert-success',
+      },
+      warning: {
+        base: 'gi-alert-warning',
+        baseDismissible: 'gi-alert-success',
+      },
+    },
+  },
+  defaultVariants: {
+    variant: 'info',
+  },
+});
+
+type AlertProps = {
+  variant?: VariantProps<typeof alertVariants>['variant'];
+  title: string;
```

Comment:
We could make the title an optional prop, sometime we may not need it?

<img width="1503" alt="image" src="https://github.com/user-attachments/assets/751a1d9b-e766-4fff-b3b4-3a7ee84e9d2d">